unit-tests/test_bip32.c

```diff
@@ -61,11 +62,10 @@ static void test_bip32_read(void **state) {
         0x00, 0x00, 0x00, 0x0a
     };
     uint32_t expected[5] = {0x8000002C, 0x80000118, 0x80000000, 0, 10};
-    uint32_t output[10] = {0};
-    bip32_path_t path_out = {.path=output, .length=0};
+    bip32_path_t path_out = { .path = {0}, .length = 0 };
```

Comment:
On previous versions `path` was a pointer to an array of `uint32_t` but since paths have a max length of 5 the type was changed to be an array of 5 elements, so the previous init method was invalid for the `bip32_path_t` type